src/include/cephfs/metrics/Types.h

```diff
@@ -78,6 +78,12 @@ struct CapInfoPayload {
     f->dump_int("cap_misses", cap_misses);
     f->dump_int("num_caps", nr_caps);
   }
+
+  void print(ostream *out) const {
+    *out << "cap_hits: " << cap_hits
+	 << "cap_misses: " << cap_misses
+	 << "num_caps: " << nr_caps;
+  }
```

Comment:
nit: if these stats are supposed to on a single line, then you probably need to add a space or comma after each item

    *out << "cap_hits: " << cap_hits
         << ", cap_misses: " << cap_misses
         << ", num_caps: " << nr_caps;

Reply:
Yeah, good catch, will fix it. Thanks @mchangir